docs/faq.md

```diff
@@ -0,0 +1,89 @@
+---
+Title: Frequently Asked Questions
+---
+
+## General Questions
+
+### Q: What is Headlamp and who is it for?
+
+**A:** Headlamp is an open source, extensible graphical user interface (GUI) designed to simplify the management of Kubernetes clusters. It serves as a visual management tool ideal for developers, DevOps professionals, system administrators, and overall Kubernetes users, who are looking for an intuitive way to oversee cluster operations, deployments, and troubleshooting.
+
+### Q: What Kubernetes flavors or vendors does Headlamp support?
+
+**A:** Headlamp is designed to be vendor-agnostic, supporting a variety of Kubernetes flavors. While it should work seamlessly with most Kubernetes distributions, we have specifically tested and ensured compatibility with certain platforms. For a comprehensive list of these tested platforms and more detailed information, please refer to our [platforms section in the documentation](https://www.headlamp.dev/docs/latest/platforms/).
+
+### Q: Is Headlamp a desktop or web application?
+
+**A:** Headlamp is available both as a desktop application and a web application. The desktop version can be installed on your local machine, providing easy access and management of your Kubernetes clusters. The web version, on the other hand, can be accessed through a browser, offering flexibility and convenience for users who prefer not to install additional software.
+
+### Q: Is there any cost involved in using Headlamp?
+
+**A:** Headlamp a 100% open source project, under the Apache 2.0 License. It is thus available at no charge, and users are encouraged to modify and redistribute it in accordance with the license terms.
+
+### Q: Can I use Headlamp for commercial purposes?
+
+**A:** Yes, Headlamp, under the Apache 2.0 License, is suitable for both personal and commercial use. This permissive license allows for the utilization of Headlamp in various settings, ranging from individual projects to large-scale enterprise operations, without any restrictions on commercial use.
+
+### Q: Where can I find the source code for Headlamp?
+
+**A:** The source code for Headlamp is publicly available on [GitHub](https://github.com/headlamp-k8s/headlamp). You are welcome to explore, fork, and contribute to the codebase.
+
+### Q: Who maintains Headlamp?
+
+**A:** Headlamp's core team is composed of developers that are employed by Microsoft. As a 100% open source project and a CNCF Sandbox project, Headlamp has a transparent governance and it encourages any users/developers to participate in it.

```

Comment:
"transparent governance" Note sure this is documented so maybe shouldn't add atm.